README.MD

```diff
@@ -1,36 +1,192 @@
+2. Docker installed
+3. [Set up discord bot](https://www.notion.so/ac75c9b08f54477ba4d8d8d20715adc3)
+4. Owned a Metamask’s wallet. If not:
+   1. [Install Metamask](https://metamask.io/download/) on your browser
+   2. After successful installation, you will be redirected to Metamask’s **Getting started** page.
+      Follow the steps to create your own wallet
 
-### Start dev
+### How to run source code locally
 
-```bash
-make dev
-```
+1. Set up source
+
+   Set up infras (db, redis), install dependencies, etc.
+
+   ```
+   make setup
+   ```
+
+2. Set up env
+
+   Update these variables inside `.env`, [How to get mnemonic](https://metamask.zendesk.com/hc/en-us/articles/360015290032-How-to-reveal-your-Secret-Recovery-Phrase)
+
+   ```
+   CENTRALIZED_WALLET_PRIVATE_KEY=“your metamask wallet mnemonic”
+   DISCORD_TOKEN='your discord bot token'
+   ```
+
+3. Run source
+
+   ```
+   make dev
+   ```
+
+   You should see the service starting with port 8200 as default
+
+### How to work on a TODO
 
-### Test
+1. Feel free to pick any TODO for you from [Board View](https://www.notion.so/2b9be3fffef74705830ad77058e35c95) → Mochi → Backlog\*\*
+2. **Assign** that item to your account
+3. Remember to update item’s **status** based on your working progress
+   - `Backlog`: not started yet
+   - `In Progress`: still working on
+   - `In Review`: Task done = PR has been merged to `develop` branch at least
+   - `Completed`: Confirmation from the team that the TODO is completely finished
+4. When switching your TODO to `In Review`, remember to fill these in
 
-```bash
-make test
+![Untitled](./images/todotime.png)
+
+### PR template
+
+```markdown
+**What does this PR do?**
+
+- [x] New API for updating ...
+- [x] Update logic of ...
+- [x] Fix error ...
+
+**How to test**
+
+- cURL
+- Steps to test new logic
+- etc.
+
+**Flowchart** (optional)
+Should have if the flow is complex
+
+**Media** (Demo) (optional)
+Attach images or demo videos
+Can insert video URL in case the size exceeds github limit
 ```
+
+## Technical Document
+
+1. **Project structure**
+
+- `cmd/` this folder contains the main application entry point files for the project,
+  - `server/main.go`: Containing init function `main()` of the service. This function will be executed when we run `make dev`
+  - Other `main.go` files (e.g. `fetch-discord-users/main.go`): Entry points to execute some functional jobs. k8s Cronjob will be set up to execute these jobs schedually
+- `docs/`: contains Swagger documentation files generated by [swaggo](https://github.com/swaggo/swag)
+- `migrations/`
+  - `schemas/`: contains DB schema migration files
+  - `seed/`: contains seed files which will initialize DB with sets of dummy data
+  - `test-seed/`: also seed files but for test DB
+- `pkg/`: contains core source code of service
+  - `cache/`: caching initial and functional methods
+  - `chain/`: crypto chains’ initial and functional methods
+  - `config/`: `.env` file’s loader and reader
```

Comment:
it's not just load and read `.env`. You should show what `config` is used for, not what it contains